### PR TITLE
[PERF-2572] - Add warmup phases to LLTMixed workload

### PR DESCRIPTION
**Pull request by MAhmadShah**
This PR encompasses the work present in this [ticket](https://jira.mongodb.org/browse/PERF-2572).

What changed:
- We have updated the LLTMixed.yml workload to include warmup phases immediately after the quiesce phases
- We have outlined the analysis for this work in this [document](https://docs.google.com/document/d/1qi8Xi_Q0b9-a5_iTH-HJOsq1aujcY3qrnSDJCS-aTU4/edit?usp=sharing)

[Patch](https://spruce.mongodb.com/version/633dc7afd6d80a510f60fd56/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC) showing successful runs. LLTMixed was run 5 times on this patch.

### CODE REVIEW

File: src/workloads/transactions/LLTMixed.yml [hunk 1]
```diff
@@ -3,15 +3,6 @@ Owner: "@mongodb/product-perf"
 Description: |
   Workload to Benchmark the effect of LongLivedTransactions on a Mixed workload.
 
-Keywords:
```
Comment (jimoleary) [inline src/workloads/transactions/LLTMixed.yml:6]: why are we deleting the keywords?

**Review by jimoleary — COMMENTED** [review]
We  should update the docs while we are here.

Can you add some patch links to show that it works?

**Comment by jimoleary** [comment]
> @jimoleary I have added a link to the patch runs in the PR description, I only ran the patch for a Linux 3-node replica set, is that sufficient?

I missed the patch bit, we only run this for the following autorun stanza, so that is perfect: 
```
      - atlas-like-replica
      - replica
      - replica-all-feature-flags
      - single-replica

```

**Comment by MAhmadShah** [comment]
> > @jimoleary I have added a link to the patch runs in the PR description, I only ran the patch for a Linux 3-node replica set, is that sufficient?
> 
> I missed the patch bit, we only run this for the following autorun stanza, so that is perfect:
> 
> ```
>       - atlas-like-replica
>       - replica
>       - replica-all-feature-flags
>       - single-replica
> ```

@jimoleary you didn't miss it, I just added it haha

**Review by jimoleary — DISMISSED** [review]
lgtm given that I have PERF-3412

**Review by brad-devlugt — DISMISSED** [review]
LGTM! Nice work on this

**Review by jimoleary — APPROVED** [review]
Thanks!

**Review by brad-devlugt — APPROVED** [review]

**Pull request merged**